configs/components/puppet.rb

```diff
@@ -14,6 +14,7 @@
   pkg.replaces 'pe-cloud-provisioner'
   pkg.replaces 'pe-puppet-enterprise-release', '4.0.0'
   pkg.replaces 'pe-agent'
+  pkg.replaces 'pe-puppetserver-common', '4.0.0'
```

Comment:
We talked about this a bit, I *think* that since it only replaces, doesn't provide then it's fine. But we'll find out :)